admin/schema.php

```diff
@@ -884,6 +884,27 @@
 
 # Release marker: 1.3.0
 
+# Count / occurrences / votes 
+$g_upgrade[210] = array( 'AddColumnSQL', array( db_get_table( 'bug' ), "
+	votes					I		NOTNULL DEFAULT '1'",
+	) );
+$g_upgrade[211] = array( 'CreateIndexSQL', array( 'idx_bug_count', db_get_table( 'bug' ), 'votes' ) );	
+
+# A standalone table is required for "alias", because not all DB allow multiple NULLs in UNIQUE index
+$g_upgrade[212] = array('CreateTableSQL', array( db_get_table( 'bug_alias' ), "
```

Comment:
Yes, it is not required. However, it is here for another reason, as indicated in the comment. I simply don't know how to add atomic operation in another way. E.g. if two applications/machines would try to post the same report (with same alias) at about the same time, Mantis should create only one report and set count to 2 (+1 on creation, +1 on second report). If you do not make this an atomic operation, you would end up with two reports with same alias - which is a big NO-NO.

I checked that Bugzilla uses transactions for that. Other implementations may use either transactions or INSERT OR UPDATE statements. I found that Mantis does not use any of those. So, introduce an unique key, try to insert, fail if it exists - it is a best try that I can do. 

If you have other suggestion how to implement atomic insert/update - you are welcome to share.